packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/index.tsx

```diff
@@ -196,6 +199,20 @@ const ConfirmSignupCode = ({
             return;
           }
         }
+        // Web integration, SubPlat redirect
+      } else if (integration.data.redirectTo) {
```

Comment:
Seems like this would be the place to make sure we are dealing with a web integration, e.g. isWebIntegration(integration)

Reply:
I didn't add that here since the web integration is the default integration so if it reaches this point that's what it should be, but I can add!